### PR TITLE
monitoring/grafana: Install a recent version of Grafana

### PR DESCRIPTION
**Pull request by aldarrie**
## Describe your changes
Added the ability to install a recent version of grafana from the Grafana project

## Checklist before requesting a review
- [x] Document introduced changes / variables in role README.md if any
- [x] Increment role version in vars/main.yml (a.b.c: +1 to b for a feature added, +1 to c for a bug fix)
- [x] Update changelog inside role README.md
- [x] If not present, please also add your name in the related collection authors list in galaxy.yml

### CODE REVIEW

File: collections/monitoring/roles/grafana/README.md [hunk 1]
```diff
@@ -140,10 +141,17 @@ If you need to install a specific version of Grafana
 ansible-playbook /etc/bluebanquise/playbooks/grafana.yml --limit management1 -e"grafana_packages_to_install='grafana-7.2.0'"
 ```
 
+Or to install a newer version which is not present in the system repositories
+```yaml
+ansible-playbook /etc/bluebanquise/playbooks/grafana.yml --limit management1 -e"grafana_install_oss_packages=true grafana_packages_to_install='grafana-9.4.7-1.x86_64.rpm'"
```
Comment (oxedions) [inline collections/monitoring/roles/grafana/README.md:146]:
```suggestion
ansible-playbook /etc/bluebanquise/playbooks/grafana.yml --limit management1 -e "grafana_install_oss_packages=true grafana_packages_to_install='grafana-9.4.7-1.x86_64.rpm'"
```

**Review by oxedions — APPROVED** [review]
Dear @aldarrie 
Many thanks for this PR 😊 
I just added a small fix proposal as I think a space is missing in the ansible-playbook command.
Everything else is ok to me, I will merge it once I have your feedback on the fix. 🙂

File: collections/monitoring/roles/grafana/README.md (hunk 1 above)
Comment (aldarrie) [reply collections/monitoring/roles/grafana/README.md]: Hello,
It's OK for me.
Thanks

**Comment by oxedions** [comment]
Dear @aldarrie 
Many thanks ! 😊

**Pull request merged**